### PR TITLE
Adding System.Net.HttpListener uapaot config

### LINKED ISSUE #16872: Add uap configuration for System.Net.HttpListener and System.Net.Mail
Those two assemblies are part of the closure of netstandard and are not includded today in the uap layout build. We need to add configurations for them.

 - [x] System.Net.Mail
 - [ ] System.Net.HttpListener

### PR DESCRIPTION
**Pull request by joperezr**
cc: @danmosemsft

Fixes #16872

Adding uapaot config to System.Net.HttpListener since it's part of the netstandard.dll closure

### CODE REVIEW

File: src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata [hunk 1]
```diff
@@ -0,0 +1,46 @@
+sspicli.dll!EncryptMessage
```
Comment (danmosemsft) [inline src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata]: I was trying to sort all these files for our merging ease (also remove any dupes but I don't see any)

**Comment by danmosemsft** [comment]
Do the tests pass with the "default" configuration against uapaot?

LGTM

File: src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata (hunk 1 above)
Comment (joperezr) [inline src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata]: in commit efe2fc3 I'm ordering and removing dupes of all of these baseline files.

**Comment by joperezr** [comment]
> Do the tests pass with the "default" configuration against uapaot?

I just tried to run the tests locally, and ilc.exe command finishes succesfully, but I'm hitting the same issue than with System.Runtime.Tests where execution halts because libc.dll is not found. This is because PlatformDetection is also used in this test assembly which adds a dependency to libc.dll. I will fix this by adding #ifdefs around the platformdetection code.

**Pull request merged**